turbo.json

```diff
@@ -1,100 +1,102 @@
 {
-    "$schema": "https://turborepo.org/schema.json",
-    "baseBranch": "origin/trunk",
-    "pipeline": {
-        "lint": {
-            "cache": false
-        },
-        "lint:fix": {
-            "cache": false
-        },
+	"$schema": "https://turborepo.org/schema.json",
+	"baseBranch": "origin/trunk",
+	"pipeline": {
+		"turbo:lint": {
+			"cache": false
+		},
 
-        "build": {
-            "dependsOn":  [ "^build", "$WC_ADMIN_PHASE" ],
-            "inputs": [
-                "src/*.js",
-                "src/**/*.js",
-                "src/*.jsx",
-                "src/**/*.jsx",
-                "src/*.ts",
-                "src/**/*.ts",
-                "src/*.tsx",
-                "src/**/*.tsx",
-                "src/*.php",
-                "src/**/*.php",
-                "includes/*.php",
-                "includes/**/*.php"
-            ],
-            "outputs": [
-                "dist/**",
-                "build/**",
-                "build-module/**",
-                "build-style/**",
-                "build-types/**"
-            ]
-        },
-        "woocommerce#build": {
-            "dependsOn":  [ 
-                "^build",
-                "woocommerce/client/admin#build",
-                "woocommerce/client/legacy#build"
-            ],
-            "outputs": [],
-            "inputs": [
-                "src/*.php",
-                "src/**/*.php",
-                "includes/*.php",
-                "includes/**/*.php"
-            ]
-        },
-        "woocommerce/client/admin#build": {
-            "dependsOn":  [ "^build", "$WC_ADMIN_PHASE" ],
-            "outputs": [],
-            "inputs": [
-                "client/*.js",
-                "client/**/*.js",
-                "client/*.jsx",
-                "client/**/*.jsx",
-                "client/*.ts",
-                "client/**/*.ts",
-                "client/*.tsx",
-                "client/**/*.tsx",
-                "client/*.scss",
-                "client/**/*.scss"
-            ]
-        },
-        
-        "test": {
-            "dependsOn":  [ "build" ],
-            "inputs": [
-                "src/*.js",
-                "src/**/*.js",
-                "src/*.jsx",
-                "src/**/*.jsx",
-                "src/*.ts",
-                "src/**/*.ts",
-                "src/*.tsx",
-                "src/**/*.tsx",
-                "src/*.php",
-                "src/**/*.php",
-                "includes/*.php",
-                "includes/**/*.php"
-            ],
-            "outputs": []
-        },
+		"lint:fix": {
```

Comment:
```suggestion
		"turbo:lint:fix": {
```

Reply:
@ObliviousHarmony interesting, is there any need for Turbo to be concerned with lint fix? it seemed a bit redundant to me because its not really something you'd want to cache I don't think? Happy to be wrong on that one 😄

Reply:
Initially, this command lived in `turbo.json` so that you could use it to fix all of the linting errors. Now that we've migrated to using `pnpm` for all commands, you're right that we should remove `lint:fix`. It may even make sense to remove `lint` altogether from the `turbo.json` file since it isn't cached and doesn't really do anything.